refinery/core/api.py

```diff
@@ -2018,6 +2019,14 @@ def obj_create(self, bundle, **kwargs):
         user = bundle.request.user
         data = json.loads(bundle.request.body)
         new_ext_group = ExtendedGroup(name=data['name'])
+
+        new_ext_group.name = new_ext_group.name.strip()
+        try:
+            new_ext_group.full_clean()
```

Comment:
There is no need to call full_clean() if there is no custom clean() method. The save() method will raise an exception if the group name is blank (a DB constraint) but it will not check for group names that contain whitespace chars. Is that the desired behavior?